testing/trino-faulttolerant-tests/src/test/java/io/trino/faulttolerant/redshift/TestRedshiftQueryFailureRecoveryTest.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.faulttolerant.redshift;
+
+import io.trino.operator.RetryPolicy;
+
+public class TestRedshiftQueryFailureRecoveryTest
```

Comment:
those should be part of `trino-redshift` module. See how we do that for mysql, postgres and others.

Reply:
fixed.